sicario.c

```diff
@@ -72,10 +72,64 @@ int kbhit(){
         return 0;
     }
 }
+
+void imprimeIntroducao();
+void inicializaTabuleiro();
```

Comment:
This and the calls before are beeing called from out of the main. Shouldn't it raise some error?

Reply:
No, this is not Python.

Reply:
Sorry my delay. I got it now. It was a misunderstood. You're right. You defines all assignatures before main and define them after. That's a good approach, congrats.